.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,120 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  test-build-release:
+
+    runs-on: ubuntu-latest
+    # Add postgres for end-to-end
+    services:
+      postgres:
+        image: postgres:10.8
+        # Set postgres env variables according to test env.yml config
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Cache pip
+        uses: actions/cache@v2
+        with:
+          # This path is specific to Ubuntu
+          path: ~/.cache/pip
+          # Look to see if there is a cache hit for the corresponding requirements file
+          key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+            ${{ runner.os }}-
```

Comment:
Have you tried without these two steps? According to the link in [your comment here](https://github.com/conveyal/transitive.js/pull/63#discussion_r593242095), Python, pip, and even node come preinstalled on Github's `ubuntu-latest`.

Reply:
Great question! I forget! (But will check.) As for removing the node install, I'm a bit on the fence about that one, since we don't necessarily want that version changing on us willy nilly.

Reply:
Looks like we don't actually need to install these! https://github.com/ibi-group/datatools-ui/pull/640/commits/bae498881934d6305eb4de2e315482441d89e92b